<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BodyConfiguratorWidgetBase</class>
 <widget class="QWidget" name="BodyConfiguratorWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="overviewGroupBox">
      <property name="title">
       <string>overview:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>1</number>
       </property>
       <property name="verticalSpacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <widget class="QTreeWidget" name="bodyTreeWidget">
         <property name="midLineWidth">
          <number>1</number>
         </property>
         <property name="columnCount">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>412</width>
         <height>395</height>
        </rect>
       </property>
       <attribute name="label">
        <string>states:</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="saveButton">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>add</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/wombat.qrc">
              <normaloff>:/icons/24/document-save.png</normaloff>:/icons/24/document-save.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="removeButton">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>remove</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/wombat.qrc">
              <normaloff>:/icons/24/edit-delete.png</normaloff>:/icons/24/edit-delete.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="loadButton">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>apply</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/wombat.qrc">
              <normaloff>:/icons/24/applications-system.png</normaloff>:/icons/24/applications-system.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QPushButton" name="seedButton">
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>use as seed</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/wombat.qrc">
              <normaloff>:/icons/24/go-jump.png</normaloff>:/icons/24/go-jump.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>13</width>
              <height>31</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
        <item>
         <widget class="QLabel" name="stateLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>96</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>412</width>
         <height>395</height>
        </rect>
       </property>
       <attribute name="label">
        <string>body parts:</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="margin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QPushButton" name="saveBodyButton">
               <property name="text">
                <string>save</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/wombat.qrc">
                 <normaloff>:/icons/24/document-save.png</normaloff>:/icons/24/document-save.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="loadBodyButton">
               <property name="text">
                <string>load</string>
               </property>
               <property name="icon">
                <iconset resource="../resources/wombat.qrc">
                 <normaloff>:/icons/24/applications-system.png</normaloff>:/icons/24/applications-system.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="bodyWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/wombat.qrc"/>
 </resources>
 <connections/>
</ui>
